transforms/dialog-full-screen-app-wrapper/README.md

```diff
@@ -0,0 +1,26 @@
+# button-destructive
+
+In order to add secondary and tertiary buttons to carbon-react, the `destructive` buttonType has been replaced with a dedicated prop.
+
+The `AppWrapper` was previously internally composed within the `DialogFullScreen` component, it has now been removed to facilitate content being aligned with the edges of the dialog.
+
+```diff
+- <DialogFullScreen {...props}>Content</DialogFullScreen>
++ <DialogFullScreen {...props}><AppWrapper>Content</AppWrapper></DialogFullScreen>
+```
+
+It's likely that `DialogFullScreen` has significant usage, therefore this codemod accounts for the various potential patterns that may have been used.
```

Comment:
I don't think there are that many use cases that are covered in this one. I think we can remove everything from the diff to the usage header.